ui/src/pages/Logs/Logs.tsx

```diff
@@ -149,7 +172,7 @@ export default function Logs() {
         containerLabelColor={containerLabelColor}
       />
       <Box sx={{ minHeight: 0, display: 'flex', flexDirection: 'column' }}>
-        <Stack direction="row" spacing={2}>
+        <Stack direction="row" spacing={2} alignContent={'center'}>
```

Comment:
Better to just do `alignContent="center"` instead of `alignContent={'center'}` to be consistent with elsewhere